docs/core/diagnostics/event-counter-perf.md

```diff
@@ -0,0 +1,100 @@
+        //      measurement in the code.
+        //   3. You don't have to have log with WriteEvent if you don't think you will ever care about details
+        //       of individual requests (that counter data is sufficient).
+
+        // This logs it to the event stream if events are on.
+        WriteEvent(1, url, elapsedMSec);
+        // This adds it to the PerfCounter called 'Request' if PerfCounters are on
+        RequestCounter.WriteMetric(elapsedMSec);
+    }
+}
+```
+
+The <xref:System.Diagnostics.Tracing.EventSource.WriteEvent%2A?displayProperty=nameWithType> line is the <xref:System.Diagnostics.Tracing.EventSource> part and is not part of <xref:System.Diagnostics.Tracing.EventCounter>, it was written to show that you can log a message together with the event counter.
+
+You logged the metric to the <xref:System.Diagnostics.Tracing.EventCounter>, but unless you can actually get the statistics out of it, it is not useful. To get the statistics, you need to enable the <xref:System.Diagnostics.Tracing.EventCounter> by setting off a timer how frequently you want the events, as well as a listener to capture the events, to do that, you can use PerfView.
+
+There is an extra keyword that you will need to specify the turn on the EventCounters.
+
+```
+PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:EventCounterIntervalSec=1 collect
+```
+
+> [!IMPORTANT]
+> The `EventCounterIntervalSec` segment is used to indicate the frequency of the sampling.
+
+As usual, turn on PerfView, and then run the sample code - we get have something like this.
+
+:::image type="content" source="media/perfview-counters.png" lightbox="media/perfview-counters.png" alt-text="PerfView of EventCounter traces":::
+
+Now let's drill into what the data captured means - when I copied from PerfView, it looks like this
+
+```
+ThreadID="17,800" ProcessorNumber="5" Payload="{ Name:"request", Mean:142.0735, StandardDeviation:42.07355, Count:2, Min:100, Max:184.1471, IntervalSec:1.000588 }"
+```
+
+Now it is obvious that within a sampling period, we have nine events, and all the other statistics.
+
+Notice that this command also logs the events, so we will get both the events and the counter statistics.
+
+:::image type="content" source="media/perfview-events.png" lightbox="media/perfview-events.png" alt-text="PerfView of event traces":::
+
+As we mentioned, to avoid overhead, sometimes we will want just the counters. This command can be used to log _only_ the counters:
+
+```
+PerfView /onlyProviders=*Samples-EventCounterDemos-Minimal:*:Critical:EventCounterIntervalSec=1 collect
+```
+
+> [!TIP]
+> The `Critical` keyword is used to filter out the other events with lower priorities.
+
+In the next release of PerfView (> 2.0.26), we can visualize the counters using PerfView. To do so, you can right-click on the event like this and choose the show event counter graph item:
```

Comment:
"next release" -- docs are supposed to avoid talking about future features ... unless "next" actually refers to something available now.